include/jemalloc/internal/witness.h

```diff
@@ -23,33 +23,34 @@
 
 #define WITNESS_RANK_BACKGROUND_THREAD_GLOBAL	4U
 
-#define WITNESS_RANK_PROF_DUMP		5U
-#define WITNESS_RANK_PROF_BT2GCTX	6U
-#define WITNESS_RANK_PROF_TDATAS	7U
-#define WITNESS_RANK_PROF_TDATA		8U
-#define WITNESS_RANK_PROF_LOG		9U
-#define WITNESS_RANK_PROF_GCTX		10U
-#define WITNESS_RANK_BACKGROUND_THREAD	11U
+#define WITNESS_RANK_PROF_RECENT_ALLOC	5U
+#define WITNESS_RANK_PROF_DUMP		6U
+#define WITNESS_RANK_PROF_BT2GCTX	7U
+#define WITNESS_RANK_PROF_TDATAS	8U
+#define WITNESS_RANK_PROF_TDATA		9U
+#define WITNESS_RANK_PROF_LOG		10U
+#define WITNESS_RANK_PROF_GCTX		11U
+#define WITNESS_RANK_BACKGROUND_THREAD	12U
```

Comment:
Any reason we cannot switch all these to an enum? It would feel a lot safer that way.

Reply:
That's a good idea. Maybe in a separate PR?